src/app/content/components/Page/PageComponent.tsx

```diff
@@ -66,6 +66,13 @@ export default class PageComponent extends Component<PagePropTypes> {
     // tslint:disable-next-line: max-line-length
     this.highlightManager = highlightManager(this.container.current, () => this.props.highlights, this.props.services, this.props.intl);
     this.scrollToTopOrHashManager = scrollToTopOrHashManager(this.container.current);
+
+    if (this.props.searchHighlights.selectedResult) {
+      this.searchHighlightManager.update({searchResults: [], selectedResult: null}, this.props.searchHighlights, {
```

Comment:
maybe we should tweak the manager to allow the prev structure to just be null

Reply:
this is the update handler so I don't know if it's worth tweaking:
```
const handleUpdate = (services: Services) => (
  previous: HighlightProp,
  current: HighlightProp,
  options: UpdateOptions
) => {
  updateResults(services, previous, current, options);
  selectResult(services, previous, current, options);
};
```

Reply:
yea you'd have to change it to be nullable all the way down, which sucks, but also manually constructing the empty state here is pretty jank, so i think that might be a good option